hintman.cabal

```diff
@@ -24,9 +24,14 @@ library
   hs-source-dirs:      src
   exposed-modules:     Hintman
                        Prelude
+                       Server
 
   build-depends:       base-noprelude
                      , relude
+                     , servant >= 0.14.1
+                     , servant-server >= 0.14.1
```

Comment:
Let's also add upper bounds for all packages. Usually in form of `servant-server ^>= 0.14.1`. To make it build better with `cabal`. But it's also a good practice in general.